docs/_data/atlantic-2024-sessions.yaml

```
@@ -0,0 +1,427 @@
+    <p>But how do docs affect RAG? Doc content, its formatting, and how it''s processed
+    all impact whether the LLM responds with knowledge and (reasonable) confidence
+    or whether the response is full of hallucinations. Come learn about RAG, how technical
+    communicators can make use of it, how our docs affect RAG performance, and what
+    we can do to make sure our users (and our LLMs) get the content they need when
+    they need it.</p>'
+- title: 'Navigating the Maze: Strategies for Successful Documentation Planning'
+  slug: navigating-the-maze-strategies-for-successful-documentation-planning-gayathri-krishnaswamy
+  series: Write the Docs Atlantic
+  series_slug: atlantic
+  year: 2024
+  speakers:
+  - name: Gayathri Krishnaswamy
+    slug: gayathri-krishnaswamy
+    twitter: 
+    website: 
+  - name: Annette Williams
+    slug: annette-williams
+    twitter: 
+    website: 
+  abstract: '<p>A documentation plan outlines an approach for creating, organizing,
+    and maintaining documentation for a software product. It serves as a blueprint
+    for the documentation process, detailing the scope, content types, user journeys,
+    priorities, and content involved in creating comprehensive documentation for the
+    product. </p>
+
+    <p>At OutSystems, until recently, we adopted a feature-driven approach without
+    a solid documentation plan for documenting product enhancements. This approach
+    was less user-centric and more product-focused, leading to limited collaboration
+    with teams like UX or training. As a result, our documentation was less comprehensive,
+    and felt disconnected from the user''s journey throughout the product.</p>
+
+    <p>Hence, early last year, for all our significant product enhancements and new
+    features, we started developing a comprehensive documentation plan that resembled
+    the real-life journey of an OutSystems user. This involved close collaboration
+    with the Research and Development (R&amp;D), UX, and training teams, which helped
+    us gain insights into our users'' needs. This increased understanding enabled
+    us to create a more user-centric documentation set that enhanced our product''s
+    usability and contributed to a positive user experience. </p>
+
+    <p>Based on our experiences, we''ll present how developing a solid documentation
+    plan enabled us to:</p>
+
+    <ul>
+
+    <li>Spend adequate time researching and mapping the user journey before creating
+    our first drafts.</li>
+
+    <li>Prioritize content delivery so the team does not face burnout and still delivers
+    the most helpful content to our users.</li>
+
+    <li>Deliver content that covers the user''s knowledge needs.</li>
+
+    </ul>
+
+    <p>The cherry on top is that we will share a usable template of our documentation
+    plan that you can readily use for your documentation projects.</p>'
+- title: 'From morphemes to manuscripts: how linguistics can make you a better writer'
```

I understand we don't force people into either title or sentence case (they can choose), but can we at least capitalize subtitles?
```suggestion
- title: 'From morphemes to manuscripts: How linguistics can make you a better writer'
```